lib/src/Uncertainty/Distribution/Trapezoidal.cxx

```diff
@@ -168,8 +168,8 @@ Complex Trapezoidal::computeCharacteristicFunction(const Scalar u) const
   Scalar real = 0.0;
   Scalar imag = 0.0;
   Complex phi;
-
-  if (std::abs(u) <= 1.0e-10) phi = Complex((-a_ + 3.0 * b_ - 3.0 * c_ + d_) * h_ / 2.0, 1.0);
+  const Scalar factor = h_ * (((c_ + d_) * (c_ * c_ * c_ + d_ * d_ * d_) + c_ * c_ * d_ * d_) - ((a_ + b_) * (a_ * a_ * a_ + b_ * b_ * b_) + a_ * a_ * b_ * b_)) / 120.0;
```

Comment:
We may inadvertently break that code without noticing the change. This cannot happen if you provide a unit test that shows the improvement in action.

Reply:
-Excellent suggestion. Done in t_Triangular_std.cxx and t_Trapezoidal_std.cxx, using Maple reference values (see validation/srd/valid_triangular_cf.mws and validation/srd/valid_trapezoidal_cf.mws)